petclinic-frontend/src/features/customers/components/AddPetForm.css

```diff
@@ -0,0 +1,133 @@
+.add-pet-form {
+  max-width: 400px;
+  margin: 20px auto;
+  padding: 20px;
+  border: 1px solid #ddd;
+  border-radius: 8px;
+  background-color: #f9f9f9;
+  box-shadow: 0 2px 8px rgba(0, 0, 0, 0.1);
+}
+
+.add-pet-form h1 {
+  text-align: center;
+  font-size: 24px;
+  color: #333;
+  margin-bottom: 20px;
+}
+
+.add-pet-form label {
+  display: block;
+  margin-bottom: 8px;
+  font-weight: bold;
+  color: #555;
+}
+
+.add-pet-form input[type='text'],
+.add-pet-form input[type='date'] {
+  width: 100%;
+  padding: 10px;
+  margin-bottom: 12px;
+  border: 1px solid #ccc;
+  border-radius: 4px;
+  transition: border-color 0.3s ease;
+}
+
+.add-pet-form input[type='text']:focus,
+.add-pet-form input[type='date']:focus {
+  border-color: #007bff;
+  outline: none;
+}
+
+.add-pet-form .error {
+  color: red;
+  font-size: 12px;
+  margin-top: -8px;
+  margin-bottom: 12px;
+}
+
+.add-pet-form button {
+  width: 100%;
+  padding: 10px;
+  border: none;
+  border-radius: 4px;
+  background-color: #28a745;
+  color: white;
+  font-size: 16px;
+  cursor: pointer;
+  transition: background-color 0.3s ease;
+}
+
+.add-pet-form button:hover {
+  background-color: #218838;
+}
+
+.add-pet-form button:disabled {
+  background-color: #ccc;
+  cursor: not-allowed;
+}
+
+.success-message {
+  margin-bottom: 20px;
+  padding: 10px;
+  background-color: #d4edda;
+  color: #155724;
+  border: 1px solid #c3e6cb;
+  border-radius: 4px;
+  text-align: center;
+}
+
+.cancel-form-button {
+  background-color: #007BFF !important;
```

Comment:
Why did you use !important here? Is there any specific reason?

Reply:
Yup, someone broke my css so I had to change that